lib/markaby/builder.rb

```diff
@@ -282,11 +255,15 @@ def initialize(*args)
       undef_method method if method_defined?(method)
     end
 
-  private
+    private
 
-    def method_missing(*args, &block)
+    def method_missing(...)
```

Comment:
FYI this syntax require Ruby >= 2.7 but the markaby gem doesn't specify [`required_ruby_version`](https://guides.rubygems.org/specification-reference/#required_ruby_version) in the gemspec ([noticed](https://github.com/sinatra/sinatra/actions/runs/7146051224/job/19462956419#step:5:23) in Sinatra CI but we've applied [workaround](https://github.com/sinatra/sinatra/pull/1965))

Reply:
What's the earliest versions that sinatra requires?  Maybe this should be fixed?

Reply:
FYI @dentarg just added required_ruby_version >= 2.7 to the gemspec and release markaby 0.9.3 https://rubygems.org/gems/markaby/versions/0.9.3

Reply:
Sinatra still supports Ruby 2.6. Nice, thanks for the very quick turnaround 👍